### PR TITLE
New announcement modal

### PR DESCRIPTION
**Pull request by dpnova**

### CODE REVIEW

**Comment by dpnova** [comment]
@fmgoncalves I had to add the SSOT tests back in, but there's a lot of code that has no coverage. Feel like helping me with some coverage for your views etc?

**Comment by fmgoncalves** [comment]
codecov is still complaining but the only uncovered diff seems to be the [local environment specific route in web/web/urls.py`](https://github.com/enhancedsociety/platform/pull/54/files#diff-b5854372922d5b23a6f7b7277faf6b6aR39).

**Comment by dpnova** [comment]
@cballantyne let's see if we can merge and deploy today.

File: web/ssot/templates/announcements.html [hunk 1]
```diff
@@ -59,12 +65,19 @@ <h1 class="h1-responsive text-center mb-4">
...
+    ann_form='{{ announcement_create_form }}'
+    action='{% url "announcements:announcements_view" project_id=project.id %}'
+    csrf_token="{% csrf_token %}">
+</announcement-create>
 
 <script src="{% static "js/moment.min.js" %}" type="text/javascript"></script>
 <script src="{% static "js/ethereumjs-tx-1.3.3.js" %}" type="text/javascript"></script>
 <script src="{% static "js/riot/riot-compiler.min.js" %}" type="text/javascript"></script>
-<script src="{% static "components/ssot-msg.tag" %}" type="riot/tag"></script>
+<script src="{% static "components/ssot-msg.tag" %}?1" type="riot/tag"></script>
```
Comment (cballantyne) [inline web/ssot/templates/announcements.html]: Is this ?1 intentional?

**Pull request merged**